### PR TITLE
[FEATURE] Implement Bonfida Twitter Handling

### PR DESCRIPTION
**Pull request by sultanpeyek**
Implement [Bonfida Name Service](https://www.npmjs.com/package/@bonfida/spl-name-service) (Twitter Handle) to show twitter username of the bidder. This feature gets good responses from the Solana eco [twitter](https://twitter.com/_Dean_Machine/status/1415905445768798210?s=20) since it could lit up the bidding wars.

### Screenshot Example
image

Feel free if you guys want to refactor the codes haha.

PS: I added `webpack-dev-server` on the package.json because I kept getting these error after adding `@bonfida/spl-name-service` package.
```
web: There might be a problem with the project dependency tree.
web: It is likely not a bug in Create React App, but something you need to fix locally.
web: The react-scripts package provided by Create React App requires a dependency:
web:   "webpack-dev-server": "3.11.0"
```

### CODE REVIEW

**Comment by bartosz-lipinski** [comment]
very nice that was one of the items on my list. 🙇‍♂️ ❤️

File: js/packages/web/src/views/auction/index.tsx [hunk 1]
```diff
@@ -202,6 +205,28 @@ const BidLine = (props: { bid: any; index: number; mint?: MintInfo, isCancelled?
...
+  // Get Twitter Handle from address
+  const connection = useConnection();
+  const [bidderTwitterHandle, setBidderTwitterHandle] = useState('');
+  useEffect(() => {
+    const getTwitterHandle = async (
+      connection: Connection,
+      bidder: PublicKey,
+    ): Promise<string | undefined> => {
+      try {
+        const [twitterHandle] = await getHandleAndRegistryKey(
```
Comment (bartosz-lipinski) [inline js/packages/web/src/views/auction/index.tsx:217]: does this one use getMultipleAccounts? I want to make sure we are not blowing up rate limits?

File: js/packages/web/src/views/auction/index.tsx [hunk 2]
```diff
@@ -202,6 +205,28 @@ const BidLine = (props: { bid: any; index: number; mint?: MintInfo, isCancelled?
...
+      } catch (err) {
+        console.warn(`err`);
+        return undefined;
+      }
+    };
+    getTwitterHandle(connection, bidder);
+  }, [bidderTwitterHandle]);
+
   return (
     <Row
```
Comment (bartosz-lipinski) [inline js/packages/web/src/views/auction/index.tsx:231]: can you also pull twitter image?

**Review by bartosz-lipinski — APPROVED** [review]

File: js/packages/web/src/views/auction/index.tsx (hunk 1 above)
Comment (sultanpeyek) [reply js/packages/web/src/views/auction/index.tsx:217]: Sorry I just read this, will check on that.

File: js/packages/web/src/views/auction/index.tsx (hunk 2 above)
Comment (sultanpeyek) [reply js/packages/web/src/views/auction/index.tsx:231]: Seems that it's not doable just by using Bonfida, but I'll try to figure it out.
Thanks @bartosz-lipinski!

File: js/packages/web/src/views/auction/index.tsx (hunk 1 above)
Comment (sultanpeyek) [reply js/packages/web/src/views/auction/index.tsx:217]: FYI @bartosz-lipinski 
https://github.com/solana-labs/solana-program-library/blob/master/name-service/js/src/twitter.ts#L245

Comment (bartosz-lipinski) [reply js/packages/web/src/views/auction/index.tsx:217]: Yeah that one is using individual getAccountInfo and doesn’t cache. We should query for all visible bids with single call and cache

**Pull request merged**